MapifyEditor/Vanilla/VanillaObject.cs

```diff
@@ -1,9 +1,12 @@
 using UnityEngine;
+using UnityEngine.Serialization;
 
 namespace Mapify.Editor
 {
     public class VanillaObject : MonoBehaviour
     {
         public VanillaAsset asset;
+        public bool keepChildren = true;
+        [FormerlySerializedAs("rotation")] public Vector3 rotationOffset = Vector3.zero;
```

Comment:
Please remove the FormerlySerializedAs attribute and ensure all prefabs are saved with the new name and the correct data.